plugins/dictionary/plugin.go

```diff
@@ -136,14 +136,9 @@ func (p *plugin) loadFile() error {
 				continue
 			}
 
-			idx := indexOf(p.definitions, line)
-			if idx > -1 {
```

Comment:
This check is performed to ensure we don't add the same definition to the list more than once. I don't know if this is a potential issue in practice. It depends on how well the dictionary.txt file is maintained.